awswrangler/opensearch/_read.py

```diff
@@ -0,0 +1,149 @@
+"""Amazon OpenSearch Read Module (PRIVATE)."""
+
+from pandasticsearch import Select, DataFrame
```

Comment:
`pandasticsearch` as a dependency worries me a bit - it is currently in 0.6.0 and hasn't updated since Nov 2020...

Reply:
agree. I've removed the dependency
@mureddy19 FYI